PENDING_PENDING_RELEASE_NOTES.md

```diff
@@ -16,6 +16,7 @@ _Date TBD_
   * /data/search_options
   * /atls
   * /acbs
+* Removed space following the colon in the /cache_status response. {"status": "OK"} becomes {"status":"OK"}
```

Comment:
Should we also do the `/status` endpoint? As well as the `/cache_status` one? Might as well, right?